### DISCUSSION TITLE
Renovate suggests upgrades that will break peer dependency requirements

### DISCUSSION THREAD

**Discussion by oliversalzburg** [renovatebot/renovate#11534] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

26.15.3

### Describe the bug

Yesterday renovate suggested that I upgrade my dependency on `graphql@15.5.1` to `graphql@15.5.2`.

After installing my dependencies locally, I saw:
```shell
╰─ yarn install                         
➤ YN0000: ┌ Resolution step
➤ YN0060: │ @fairmanager/core-api@workspace:packages/core-api provides graphql (p69b79) with version 15.5.2, which doesn't satisfy what graphql-relay requests
➤ YN0060: │ @fairmanager/core-graphql@workspace:packages/core-graphql provides graphql (p8c9a7) with version 15.5.2, which doesn't satisfy what graphql-relay requests
➤ YN0060: │ @fairmanager/core-models3@workspace:packages/core-models3 provides graphql (p0f726) with version 15.5.2, which doesn't satisfy what graphql-relay requests
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 0s 298ms
```
Further looking into the problem:
```shell
╰─ yarn explain peer-requirements p69b79      
➤ YN0000: @fairmanager/core-api@workspace:packages/core-api provides graphql@npm:15.5.2 with version 15.5.2, which doesn't satisfy the following requirements:

➤ YN0000: graphql-relay@npm:0.8.0 [0480e] → 15.5.1 ✘
```

So `graphql-relay` wants `graphql@15.5.1` specifically, which I satisfied. The renovate upgrade breaks this dependency chain.

I feel like this shouldn't happen. Did I forget to configure something?

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
renovate didn't track / validate any transitiv dependencies. This is by design

**Reply by rarkins** [reply]
If we suppressed a PR because its peer dependencies are no longer met then you would often end up with gridlock where the update is stuck forever. You're better off getting the PR, having your package manager fail it, and deciding whether to ignore the update or makes the changes necessary.

**Reply by oliversalzburg** [reply]
Fair enough. I'll look for a solution elsewhere. Thanks for the quick support! 😊

**Reply by oliversalzburg** [reply]
For completeness, I ended up solving this for me by using a log filter (`.yarnrc.yml`) in yarn to upgrade the warning to an error:
```yml
logFilters:
  - code: YN0060
    level: error
```
